data_summaries/AgentInstruct.json

```diff
@@ -0,0 +1,301 @@
+        "Paper Title": "AgentBench: Evaluating LLMs as Agents",
+        "Dataset URL": "https://huggingface.co/datasets/THUDM/AgentInstruct/viewer/default/kg",
+        "GitHub URL": "https://github.com/THUDM/AgentBench",
+        "Hugging Face URL": "https://huggingface.co/datasets/THUDM/AgentInstruct/viewer/default/kg",
+        "Papers with Code URL": "",
+        "ArXiv URL": "https://arxiv.org/abs/2310.12823",
+        "Semantic Scholar Corpus ID": 260682249,
+        "Collection": "AgentInstruct",
+        "Collection URL": "https://huggingface.co/datasets/THUDM/AgentInstruct",
+        "Languages": [
+            "English"
+        ],
+        "Task Categories": [
+            "Chain-of-Thought",
+            "Instruction Following"
+        ],
+        "Text Sources": [
+            "freebase"
+        ],
+        "Model Generated": [],
+        "Format": [
+            "Multi-turn Dialog"
+        ],
+        "Human Annotation": "Yes",
+        "Derived from Datasets": [
+            "GrailQA",
+            "ComplexWebQuestions",
+            "GraphQuestions"
+        ],
+        "Creators": [
+            "Tsinghua University",
+            "The Ohio State University",
+            "UC Berkeley"
+        ],
+        "Licenses": [
+            {
+                "License": "Unspecified",
+                "License URL": ""
+            }
+        ],
+        "License Notes": "The paper cites an anonymous source from 2023 that is currently under review. It is unclear what that means for the license. The original Freebase dataset, which seems to be a part of the source, is licensed under the Creative Commmon Attribution License.",
+        "License Verified By": "Christopher",
+        "Dataset Filter IDs": [
+            "kg"
+        ]
+    },
+    "AgentInstruct-operatingsystem": {
+        "Unique Dataset Identifier": "AgentInstruct-operatingsystem",
+        "Dataset Name": "operatingsystem",
+        "Paper Title": "AgentBench: Evaluating LLMs as Agents",
+        "Dataset URL": "https://huggingface.co/datasets/THUDM/AgentInstruct/viewer/default/os",
+        "GitHub URL": "https://github.com/THUDM/AgentBench",
+        "Hugging Face URL": "https://huggingface.co/datasets/THUDM/AgentInstruct/viewer/default/os",
+        "Papers with Code URL": "",
+        "ArXiv URL": "https://arxiv.org/abs/2310.12823",
+        "Semantic Scholar Corpus ID": 260682249,
+        "Collection": "AgentInstruct",
+        "Collection URL": "https://huggingface.co/datasets/THUDM/AgentInstruct",
+        "Languages": [
+            "English"
+        ],
+        "Task Categories": [
+            "Instruction Following",
+            "Chain-of-Thought"
+        ],
+        "Text Sources": [
+            "stackoverflow.com"
+        ],
+        "Model Generated": [
+            "OpenAI GPT-4"
+        ],
+        "Format": [
+            "Multi-turn Dialog"
+        ],
+        "Human Annotation": "Yes",
+        "Derived from Datasets": [],
```

Comment:
AgentBench? (It's complex so maybe I'm misunderstanding it, feel free to override)

Reply:
you are absolutely right - that is actually a part of the AgentBench dataset/environment